### PR TITLE
Rename deprecated macros

### PR DESCRIPTION
**Pull request by yantangx**
rename deprecated definition and parameters

Issue: N/A
Test: N/A
Signed-off-by: Ethan Wang <ethan.wang@intel.com>

### CODE REVIEW

File: _studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h [hunk 1]
```diff
@@ -97,7 +97,7 @@ namespace MfxHwH264Encode
     static const mfxU8 SEI_TYPE_MVC_SCALABLE_NESTING           = 37;
 // MVC BD }
 
-    static const mfxU16 MFX_RATECONTROL_WIDI_VBR = 100;
+    static const mfxU16 MFX_RATECONTROL_Wireless_VBR = 100;
```
Comment (dmitryermilov) [inline _studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h:100]: In capital, please

File: _studio/shared/include/mfx_utils.h [hunk 2]
```diff
@@ -145,7 +145,11 @@ bool LumaIsNull(const mfxFrameSurface1 * surf)
 
 
 #ifdef MFX_ENABLE_CPLIB
-    #define IS_PROTECTION_CENC(val) (MFX_PROTECTION_CENC_WV_CLASSIC == (val) || MFX_PROTECTION_CENC_WV_GOOGLE_DASH == (val))
+    enum {
+        MFX_PROTECTION_CENC_ANDROID_WV_CLASSIC      = 0x0004, /*!< The protection scheme is based on the Hardware DRM*. */
+        MFX_PROTECTION_CENC_ANDROID_WV_DASH         = 0x0005, /*!< The protection scheme is based on the Modular DRM*. */
+    };
```
Comment (dmitryermilov) [inline _studio/shared/include/mfx_utils.h:151]: Please use definitions from public headers

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: _studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h (hunk 1 above)
Comment (yantangx) [reply _studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h:100]: > In capital, please

sure

File: _studio/shared/include/mfx_utils.h (hunk 2 above)
Comment (yantangx) [reply _studio/shared/include/mfx_utils.h:151]: > Please use definitions from public headers

sure, and this part will also be mark as false hit in IPLDT scan

**Comment by yantangx** [comment]
this change is not needed. Reviewed IPLDT scan result, agreed to mark “GOOGLE” “Widevine” “PAVP” “decrypt” “Encrypted” as false hit.

**Pull request closed without merging**